docset/winserver2022-ps/ServerManager/Get-WindowsFeature.md

```diff
@@ -11,70 +11,93 @@ title: Get-WindowsFeature
 # Get-WindowsFeature
 
 ## SYNOPSIS
-Gets information about Windows Server roles, role services, and features that are available for installation and installed on a specified server.
+Gets information about Windows Server roles, role services, and features that are available for
+installation and installed on a specified server.
 
 ## SYNTAX
 
 ```
-Get-WindowsFeature [[-Name] <String[]>] [-Vhd <String>] [-ComputerName <String>] [-Credential <PSCredential>]
- [-LogPath <String>] [<CommonParameters>]
+Get-WindowsFeature [[-Name] <String[]>] [-Vhd <String>] [-ComputerName <String>]
+[-Credential <PSCredential>] [-LogPath <String>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The `Get-WindowsFeature` cmdlet gets information about features that are both available for installation and already installed on a computer that is running Windows Server or an offline virtual hard disk (VHD) that is running Windows Server.
+
+The `Get-WindowsFeature` cmdlet gets information about features that are both available for
+installation and already installed on a computer that is running Windows Server or an offline
+virtual hard disk (VHD) that is running Windows Server.
 
 ## EXAMPLES
 
 ### Example 1
+
 ```powershell
 Get-WindowsFeature -ComputerName Server1 -Credential contoso.com\user1
 ```
 
-This example gets a list of features that is available and installed on the target computer named Server1.
-The credentials for user user1 in the Contoso.com domain, a user who has Administrator rights on Server1, are provided.
+This example gets a list of features that is available and installed on the
+target computer named Server1.
+The credentials for user user1 in the Contoso.com domain, a user who has
+Administrator rights on Server1, are provided.
 
 ### Example 2
+
 ```powershell
 Get-WindowsFeature -Vhd D:\ps-test\vhd1.vhd
 ```
 
-This example returns a list of features that is available and installed on the specified offline VHD located at D:\ps-test\vhd1.vhd.
+This example returns a list of features that is available and installed on the specified offline VHD
+located at D:\ps-test\vhd1.vhd.
```

Comment:
We always want to format filepaths as code so they display correctly and so that internationalization doesn't treat them like normal words.

```suggestion
located at `D:\ps-test\vhd1.vhd`.
```